theano/compile/debugmode.py

```diff
@@ -613,44 +613,10 @@ def _optcheck_fgraph(input_specs, output_specs, accept_inplace=False):
         instances already installed.
 
     """
-    orig_inputs = [spec.variable for spec in input_specs]
     updates = [spec.update for spec in input_specs if spec.update]
```

Comment:
delete this line and return from this function like this:

fgraph, updates, equivalence_tracker